### PR TITLE
Update DQC's README.md with PR standards.

### PR DESCRIPTION
**Pull request by chrislococo-wf**
#### Changes:
- Added `dist/` to the `.gitignore` file in order to prevent confusion with that directory when building the library locally and to prevent it accidentally being added.
- Updated the names of the requirements files to align better with convention.
- Updated the `README.md` with pull request standards and a standard PR template.

View of the updated readme: https://github.com/chrislococo-wf/dqc_us_rules/blob/XS-2855/README.md#requirements-for-a-pull-requestpr
Please review: @hefischer  @andrewperkins-wf
FYI: @susanyount-wf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +61,28 @@ See documentation in the test suite
 
 The rule definition index is [here](docs/README.md).
 
-## Change Management
+## Proposed Changes
+
+We actively accept, and encourage, pull requests for code changes.  An list of the requirements for a pull request follows, and the request will be reviewed by the technical leads of the project.  If the request is accepted it will be merged into the appropriate branch. Some requests may require Committee approval which may take longer to implement.  If the request is found to be missing parts or is otherwise incomplete, comments will be noted regarding the missing or incomplete parts.
```
Comment (andrewperkins-wf) [inline README.md]: `A list` instead of `An list`

**Comment by andrewperkins-wf** [comment]
pass complete

**Comment by andrewperkins-wf** [comment]
+1

**Comment by brianmyers-wf** [comment]
+1

File: README.md [hunk 2]
```diff
@@ -65,9 +61,28 @@ See documentation in the test suite
...
+
+  - Branch off master, develop on your independent fork, PR back to master or other appropriate branch on the root fork.
+  - Your code should pass [flake8](https://github.com/bmcustodio/flake8).
+  - Unit test coverage is required or an explanation for why the change is already covered or not coverable.
+  - Good [Docstrings](https://github.com/Workiva/styleguide/blob/master/PYTHON.rst#docstrings) are required.
+  - Good [commit messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) are required.
+  - The pull request must have at least one `+1` from another community member.
+  - There must be no unaddressed comments.
+  - PR text must be in the specified format:
+    ```
```
Comment (brianmyers-wf) [inline README.md:78]: It looks like you can actually have a file named `PULL_REQUEST_TEMPLATE` in the root of the repo to have it auto template? maybe we should do that here?

Comment (chrislococo-wf) [inline README.md:78]: Yeah we can look into it for a follow up. I wanted to get the template out there ASAP, but I will experiment with the auto-PR formatting as well and see if I can cut a followup PR.

**Comment by mindeerickson-wf** [comment]
Spoke with Chris...no QA needed for this ticket.  

QA +1

**Pull request merged**